src/ralph/utils.py

```diff
@@ -84,3 +84,16 @@ def now():
     """Return the current UTC time in ISO format"""
 
     return datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+
+
+def nested_get(dictionary, keys):
+    """Get the nested dict value by keys array."""
+
+    if keys is None:
+        return None
+    for key in keys:
+        try:
+            dictionary = dictionary[key]
+        except (KeyError, TypeError):
```

Comment:
```suggestion
        except KeyError:
```

Reply:
I have added the `TypeError` in case the `dictionary` is not  a `dict`. But now, with the type hints, I agree that this is not needed anymore. Thanks)

Reply:
I have put back the `TypeError`, this happens in case we try to access fields that are not present in the `dict_`.
Example: `keys="a.b.c"` and `dict_={"a": "b"}`, then `nested_get` will try to access `"b"["c"]` which raises a `TypeError`